examples/headless-demo/index.html

```diff
@@ -97,18 +153,19 @@
 
   <script>
     function generate() {
-      // Parse the XML into a tree.
-      var xmlText = document.getElementById('xml_input').value;
+      // Parse the JSON into a tree.
+      var jsonText = document.getElementById('json_input').value;
+      var code;
       try {
-        var xml = Blockly.Xml.textToDom(xmlText)
+        var json = JSON.parse(jsonText)
+        var demoWorkspace = new Blockly.Workspace();
+        Blockly.serialization.workspaces.load(json, demoWorkspace);
+        code = Blockly.Python.workspaceToCode(demoWorkspace);
```

Comment:
I don't think these have the ability to throw. Did you run into issues, or just moving these to be extra cautious?

Reply:
I wasn't sure. I've left the `JSON.parse()` in.